app/src/main/res/values/strings.xml

```diff
@@ -96,6 +96,7 @@
   <string name="download_play">Play</string>
```

Comment:
Right-click `download_play` and select `Find Usages`, you will find that there are none so can delete this line.

Reply:
@abdulwd done.